### DISCUSSION TITLE
Missing sources files in the CMakeLists.txt

### DISCUSSION THREAD

**Discussion by rhuelga** [mbitsnbites/vasm-mirror#17] [General]
Following the instructions on linux with cmake, to build the m68k with mot. I get missing symbols error. I have to add some files (source.c, listing.c, output_ihex.c, output_cdef.c, output_o65.c) to the CMakeLists.txt to solve them and compile.

Because it is a mirror, I didn't create a pull request. And I tried to get access the main page of the project [sun.hasenbraten.de/vasm](http://sun.hasenbraten.de/vasm/), but it looks like the server is down.

I hope this will help somebody.

`diff --git a/CMakeLists.txt b/CMakeLists.txt
index 7665b4e..f3340c5 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,6 +7,8 @@ set(VASM_SYNTAX "std" CACHE STRING "vasm assembler syntax")
 # vasm
 set(vasm_sources
     vasm.c
+    source.c
+    listing.c
     atom.c
     expr.c
     symtab.c
@@ -23,6 +25,9 @@ set(vasm_sources
     syntax/${VASM_SYNTAX}/syntax.c
     output_test.c
     output_elf.c
+    output_ihex.c
+    output_cdef.c
+    output_o65.c
     output_bin.c
     output_vobj.c
     output_hunk.c
@@ -57,4 +62,3 @@ target_include_directories(
     cpus/${VASM_CPU}
     syntax/${VASM_SYNTAX}
     )
-
`

**Comment by StarWolf3000** [comment]
For maybe the same reason, building using CMake never generated usable binaries on Windows (always complained about missing output modules), hence why I always use the GNU Make compilation steps and those worked fine for me.
